### PR TITLE
Add initial installation page to website.

### PR DESCRIPTION
**Pull request by vext01**
This stubs out an install page and adds install instructions for OpenBSD.

Raising as a draft, as other packagers need to add their install instructions.

Link to it is in a prominent position on the front page, as it's probably the first thing you want users to do: install it!

image

### CODE REVIEW

File: content/posts/install/index.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+"ShowToc" = true
+"TocOpen" = false
+"weight" = 2
++++
+
+## Via package managers
+
+The easiest way to get Rizin and Cutter is to install from an operating system
+package manager.
+
```
Comment (wargio) [inline content/posts/install/index.md:17]: Probably worth mentioning OBS?
@ret2libc

Comment (vext01) [reply content/posts/install/index.md:17]: The allow edits by maintainers box is checked, so please push a commit :)

**Comment by vext01** [comment]
Just noticed that the buttons on the landing page now look odd, since one is a different colour to the other two. Shall we make them all the light purple?

**Comment by ITAYC0HEN** [comment]
Thanks for the contribution! :)
Generally speaking, Given that the <kbd>Documentation</kbd> button is highly visible and centric, I don't find a reason to have an additional Install page. The Building page is the second one on the documentation.
If we want to describe installation via package managers I suggest improving the documentation instead by adding an "Download and Install" page, or modify the documentation front page with quick links to common actions (download, install, build).

In case the team still prefers having an Installation page on the website then anyway I don't think it should be a button, but instead, either add a to-level page to the top menu or modify one of the homepage content blocks if one of them is irrelevant now.

**Comment by vext01** [comment]
> Given that the Documentation button is highly visible and centric, I don't find a reason to have an additional Install page.

OK let's simulate a new user, who just wants to quickly try out rizin.

 - goes to `rizin.re`.
 - This looks cool.OK, how do I get this?
 - Scans the links at the top of the page.
    - Source code: meh, can't be bothered to build stuff.
    - Documentation: meh, don't want to read stuff just yet.
 - Hopefully doesn't give up.

Let's make rizin as easy as possible to get on people's systems.

I didn't know the docs has a "downloading" page. I think a big button on the front page should link to that, and we can list distros in that page.

Sounds good?

**Comment by XVilka** [comment]
I personally agree - installation page should be linked somewhere from the main page on a visible place.

**Comment by ITAYC0HEN** [comment]
I see no problem with _replacing_ the <kbd>Source</kbd> button with a <kbd>Download</kbd> or <kbd>Install</kbd> button, and adding "Github" or "Source" item to the top menu.

**Comment by wargio** [comment]
i actually think we could just have that, so source top bar and download center of the screen.

**Comment by ITAYC0HEN** [comment]
Fine by me, but not a triple-buttons :)
And re: the content of the download install page - up to you. I support linking to a single-maintained place, instead of maintaining duplicate pages that sooner or late will become asynced

**Comment by ret2libc** [comment]
@vext01 are you going to update this? If not, do you mind if I pick it up and apply some further changes on top of this?

**Comment by vext01** [comment]
Please go ahead, as I'm a little short on time at the moment.

**Comment by ret2libc** [comment]
image
image

Please note the "Source" link in the top bar. The install button is there on the main page and it is not a blog post but just a regular page.

**Comment by wargio** [comment]
i like it

**Review by XVilka — APPROVED** [review]

**Review by wargio — APPROVED** [review]

**Pull request merged**